src/Domain/Donation.php

```diff
@@ -215,14 +215,14 @@ public function toHookModel(): array
         $data['updatedTime'] = $this->getUpdatedDate()->format(DateTime::ATOM);
         $data['amountMatchedByChampionFunds'] = (float) $this->getConfirmedChampionWithdrawalTotal();
         $data['amountMatchedByPledges'] = (float) $this->getConfirmedPledgeWithdrawalTotal();
+        $data['optInCharityEmail'] = $this->getCharityComms();
```

Comment:
@tbg-jasonfung this is already set this way in `toApiModel()`, so if this is the solution then we don't need the double-set – deleting the `unset` line has the same effect.